src/main/java/com/example/ShortnerURL/WebDto/ErrorResponseDto.java

```diff
@@ -0,0 +1,12 @@
+package com.example.ShortnerURL.WebDto;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Value;
+
+@Value
+@Getter
+@Builder
+public class ErrorResponseDto {
```

Comment:
this response can be used across all apis, via an `@ExceptionHandler(...)` should a response of this DTO.